docker-make

```diff
@@ -3,4 +3,4 @@
 set -ex
 
 sudo docker build -t mattermost-plugin-jitsi-builder .
-sudo docker run --rm -it -v $(pwd):/src -w /src mattermost-plugin-jitsi-builder make "$@"
\ No newline at end of file
+sudo docker run --rm -it -v $(pwd):/src -w /src mattermost-plugin-jitsi-builder make "$@"
```

Comment:
It may make sense to have this in `build/custom.mk` to make it into a `make` command, like `make build-docker`. 0/5